ush/python/pygfs/task/atm_analysis.py

```diff
@@ -0,0 +1,439 @@
+
+        # atm DA needs coupler
+        basename = f'{to_fv3time(task_config.current_cycle)}.coupler.res'
+        bkglist.append([os.path.join(rst_dir, basename), os.path.join(run_dir, basename)])
+
+        # atm DA needs core, srf_wnd, tracer, phy_data, sfc_data
+        for ftype in ['core', 'srf_wnd', 'tracer']:
+            template = f'{to_fv3time(self.task_config.current_cycle)}.fv_{ftype}.res.tile{{tilenum}}.nc'
+            for itile in range(1, task_config.ntiles + 1):
+                basename = template.format(tilenum=itile)
+                bkglist.append([os.path.join(rst_dir, basename), os.path.join(run_dir, basename)])
+
+        for ftype in ['phy_data', 'sfc_data']:
+            template = f'{to_fv3time(self.task_config.current_cycle)}.{ftype}.tile{{tilenum}}.nc'
+            for itile in range(1, task_config.ntiles + 1):
+                basename = template.format(tilenum=itile)
+                bkglist.append([os.path.join(rst_dir, basename), os.path.join(run_dir, basename)])
+
+        bkg_dict = {
+            'mkdir': [run_dir],
+            'copy': bkglist,
+        }
+        return bkg_dict
+
+    @logit(logger)
+    def get_berror_dict(self, config: Dict[str, Any]) -> Dict[str, List[str]]:
+        """Compile a dictionary of background error files to copy
+
+        This method will construct a dictionary of either bump of gsibec background
+        error files for global atm DA and return said dictionary for use by the
+        FileHandler class.
+
+        Parameters
+        ----------
+        config: Dict
+            a dictionary containing all of the configuration needed
+
+        Returns
+        ----------
+        berror_dict: Dict
+            a dictionary containing the list of atm background error files to copy for FileHandler
+        """
+        SUPPORTED_BERROR_STATIC_MAP = {'identity': self._get_berror_dict_identity,
+                                       'bump': self._get_berror_dict_bump,
+                                       'gsibec': self._get_berror_dict_gsibec}
+
+        try:
+            berror_dict = SUPPORTED_BERROR_STATIC_MAP[config.STATICB_TYPE](config)
+        except KeyError:
+            raise KeyError(f"{config.STATICB_TYPE} is not a supported background error type.\n" +
+                           f"Currently supported background error types are:\n" +
+                           f'{" | ".join(SUPPORTED_BERROR_STATIC_MAP.keys())}')
+
+        return berror_dict
+
+    @staticmethod
+    @logit(logger)
+    def _get_berror_dict_identity(config: Dict[str, Any]) -> Dict[str, List[str]]:
+        """Placeholder for atm identity background error files to copy
+
+        This method currently does nothing since the identity background error
+        does not require any external files.
+
+        This is a private method and should not be accessed directly.
+
+        """
+        logger.info(f"Identity background error does not use staged files.  Return empty dictionary")
```

Comment:
I am guessing that we actually need to return an empty dict though, and not just say it in the logger.
```
return {}
```

Reply:
Change committed at [9d3d147](https://github.com/RussTreadon-NOAA/global-workflow/commit/9d3d147639e70646d8daf1164b463497d1cafc66).

Reply:
[e6fbc44](https://github.com/NOAA-EMC/global-workflow/pull/1372/commits/e6fbc44c067c3f3dd5bc79dfe0228424126e6c08) looks good to me.